gatsby-config.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
       options: {
         name: 'content',
         path: './content/',
+        ignore: ['**/_archiv/*'],
```

Comment:
Hatte neulich die neuen JSONs (20200716) in den content-Ordner gelegt und die alten in `_archiv` verschoben und darin nochmal eine Untergruppierung nach Monat. Leider hat das Gatsby-Filesystem-Plugin Probleme mit Ordnernamen, die rein aus Zahlenfolgen bestehen.
Lasse deshalb den `_archiv`-Ordner über diese Zeile komplett ignoriert.